watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/metab/metab.py

```diff
@@ -480,42 +480,6 @@ def rule_LCOCR_comp(b, c):
     m.fs.costing.LCOCR_comp = Expression(m.fs.costing.LC_comp, rule=rule_LCOCR_comp)
 
 
-def adjust_default_parameters(m):
-    m.fs.metab_hydrogen.hydraulic_retention_time.fix(6)  # default - 12 hours, 0.5x
```

Comment:
@TimBartholomew Is it fine to delete this function. Codecov was failing because of this, and it was apparently only used in the `metab_ui.py` file, which has been deleted.